documentation/modules/proc-configuring-kafka-user.adoc

```diff
@@ -16,7 +16,7 @@ For example:
 
 When you configure the `KafkaUser` authentication and authorization mechanisms, ensure they match the equivalent `Kafka` configuration:
 
-* `KafkaUser.spec.authentication` matches `Kafka.spec.kafka.listeners.*.authentication`
+* `KafkaUser.spec.authentication` matches `Kafka.spec.kafka.listeners.authentication`
```

Comment:
the `authentication` field is not right under the `listeners` one because you can specify different authentication for every listener. I don't know in this case what's the best notation, maybe something like:

`Kafka.spec.kafka.listeners.<listener>.authentication`

@scholzj wdyt?

Reply:
Well, I guess technically it would be like this: `Kafka.spec.kafka.listeners[*].authentication`, or?

Reply:
yeah, agree, using the JSON path notation is better.

Reply:
Thanks. Fixed in three places.